migrations/Version202311171021031888_taoMediaManager.php

```
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoMediaManager\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\tao\scripts\update\OntologyUpdater;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ *
+ * phpcs:disable Squiz.Classes.ValidClassName
+ */
+final class Version202311171021031888_taoMediaManager extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Changed http://www.tao.lu/Ontologies/TAOMedia.rdf#Language is_language_dependent to False';
+    }
+
+    public function up(Schema $schema): void
+    {
+        OntologyUpdater::syncModels();
+    }
+
+    public function down(Schema $schema): void
+    {
+       // do nothing
```

Maybe throw this exception here as it cannot be reverted:
```
        throw new IrreversibleMigration('Reverting this migration does not make sense');
```